script/hassfest/quality_scale.py

```diff
@@ -1133,6 +1133,7 @@ class Rule:
     "wsdot",
     "wyoming",
     "x10",
+    "xai_conversation",
```

Comment:
Please add a quality scale file instead. New integrations require Bronze tier:

https://developers.home-assistant.io/docs/core/integration-quality-scale/

https://developers.home-assistant.io/docs/core/integration-quality-scale/#keeping-track-of-the-implemented-rules